packages/dapp/src/components/newsroom/NewsroomManagementV2.tsx

```diff
@@ -28,6 +33,7 @@ export default class NewsroomManagement extends React.Component<NewsroomManageme
         address={this.props.match.params.newsroomAddress}
         theme={DEFAULT_BUTTON_THEME}
         metamaskEnabled={this.state.metamaskEnabled}
+        allSteps={true}
```

Comment:
what is this for?

Reply:
Somewhat poorly named but couldn't think of a better one and comment describes the prop in the newsroom component - it's for adding sign constitution and full apply to TCR step, so that both can be used for testing in the dapp, while excluding them from wordpress plugin.